src/js/tabs/receiver.js

```diff
@@ -4,13 +4,40 @@ const receiver = {
     rateChartHeight: 117,
     analyticsChanges: {},
     needReboot: false,
+    elrsPassphraseEnabled: false,
 };
 
 receiver.initialize = function (callback) {
     const tab = this;
 
-    if (GUI.active_tab !== 'receiver') {
-        GUI.active_tab = 'receiver';
+    GUI.active_tab = 'receiver';
+
+    function lookup_elrs_passphrase(uidString) {
+        const passphraseMap = ConfigStorage.get('passphrase_map').passphrase_map || {};
+
+        return passphraseMap[uidString] ?? 0;
+    }
+
+    function save_elrs_passphrase(uidString, passphrase) {
+        const passphraseMap = ConfigStorage.get('passphrase_map').passphrase_map ?? {};
+
+        passphraseMap[uidString] = passphrase;
+        ConfigStorage.set({'passphrase_map': passphraseMap});
+      }
+
+    function elrs_passphrase_to_bytes(text) {
+        const MD5 = require('md5.js');
```

Comment:
to prevent a possible memory leak, move this line up to line #11.
This is still "local" but for sure only gets called once (receiver.init).